blog/2023-07-23-openhab-4-0-release.md

```diff
@@ -0,0 +1,148 @@
+---
+layout: BlogPost
+category: blog
+title: openHAB 4.0 Release
+author: Kai Kreuzer and others
+date: '2023-07-23T18:00:00+02:00'
+previewimage: /uploads/summer23.jpg
+tags:
+  - news
+excerpt: >-
+  A new major release has arrived! openHAB 4 brings some exciting new features and several improvements for our users as well as our developers!
+---
+A new major release has arrived! openHAB 4 brings some exciting new features and several improvements for our users as well as our developers!
+
+With openHAB4, we have finally reached feature parity between UI-based and file-based configurations with UI support for both transformation and persistence configuration, added support for using scripts as transformations and profiles, and levelled up Units of Measurement (UoM) support!
```

Comment:
Maybe somehow add that scripts as transformations are available for all languages, nit just JS.